test/integration/content-preview/Annotations.e2e.test.js

```diff
@@ -0,0 +1,89 @@
+import('https://unpkg.com/box-annotations/dist/annotations.js');
+
+describe('ContentPreview with Annotations', () => {
+    const helpers = {
+        load({ features, fileId, props } = {}) {
+            cy.visit('/Elements/ContentPreview', {
+                onBeforeLoad: contentWindow => {
+                    contentWindow.FEATURES = {
+                        activityFeed: {
+                            annotations: {
+                                enabled: true,
+                            },
+                        },
+                        ...features,
+                    };
+                    contentWindow.FILE_ID = fileId;
+                    contentWindow.PROPS = {
+                        showAnnotations: true,
+                        ...props,
+                    };
+
+                    const { document } = contentWindow;
+                    const { head } = document;
+
+                    const link = document.createElement('link');
+                    link.rel = 'stylesheet';
+                    link.type = 'text/css';
+                    link.href = 'https://unpkg.com/box-annotations/dist/annotations.css';
+                    head.appendChild(link);
+                },
+            });
+        },
+    };
+
+    it('Should scroll to annotations', () => {
+        helpers.load({
+            fileId: Cypress.env('FILE_ID_DOC_ANNOTATIONS'),
+            props: {
+                boxAnnotations: new global.BoxAnnotations(null, { features: { highlightText: true } }),
+            },
+        });
+
+        cy.get('.ba-RegionAnnotation').should('be.visible'); // Region target is on the first page
+        cy.get('.ba-HighlightTarget-rect').should('not.be.visible'); // Highlight target is on the second page
+
+        cy.contains('Highlight Annotation on version 2')
+            .siblings()
+            .filter('button')
+            .contains('Page 2')
+            .click();
+
+        cy.get('.ba-RegionAnnotation').should('not.be.visible');
+        cy.get('.ba-HighlightTarget-rect').should('be.visible');
+
+        cy.contains('Region Annotation on version 2')
+            .siblings()
+            .filter('button')
+            .contains('Page 1')
+            .click();
```

Comment:
Could these be turned into a helper function? Something like `clickAnnotationLinkByComment` perhaps? Is validating the link's text part of the test or could we do `filter` by a `data-testid`?